.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+## Proposed changes
+
+Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.\
+If it fixes a bug or resolves a feature request, be sure to link to that issue.
+
+## Types of changes
+
+What types of changes does your code introduce to the php Github AApi?
+_Put an `x` in the boxe(s) that apply_
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] Improvement (non-breaking change which improves an existing code/behavior)
+- [ ] Deprecated third party dependency update
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation/Typo/Resource/Annotation update that does not involve any direct code modification
+- [ ] Other, please specify: 
```

Comment:
Will this really help anyone or just slow down contributors?

Reply:
Mostly it allows maintainer to identify quickly the kind of issue he's dealing with. Remember that this is rendered as a checkbox which is easily clickable.

Reply:
I understand. But you are fixing a problem this repository does not have :)

Reply:
Sure, but it's not about fixing a problem in the repo, it's being more organized which will surely be useful in the future if the library is going to grow quickly :)
But if you think this is not adding a real value at all I can remove it ofc.